src/Common/CSharp/Analysis/AddExceptionToDocumentationComment/AddExceptionToDocumentationCommentAnalysis.cs

```diff
@@ -216,4 +220,37 @@ private static bool InheritsFromException(ITypeSymbol typeSymbol, INamedTypeSymb
             && typeSymbol.BaseType?.IsObject() == false
             && typeSymbol.InheritsFrom(exceptionSymbol);
     }
+
+    private static bool IsExceptionTypeCaughtByCatch(TryStatementSyntax tryStatementSyntax, IdentifierNameSyntax exceptionType)
+        => tryStatementSyntax.Catches.Any(x => x.Declaration?.Type is IdentifierNameSyntax ins1 && ins1.Identifier.Value == exceptionType.Identifier.Value);
+
+    /// <summary>
+    /// Walk upwards from throw statement and find all try statements in method and see if any of them catches the thrown exception type
+    /// </summary>
+    private static bool IsExceptionTypeCaughtInMethod(SyntaxNode node, ExpressionSyntax expressionSyntax)
+    {
+        if (expressionSyntax is not ObjectCreationExpressionSyntax x || x.Type is not IdentifierNameSyntax exceptionType)
```

Comment:
I now instead take the INamedTypeSymbol for the exception directly, since we'd already computed it outside of this method.

That meant that this entire section was removed.